src/components/CRUDDash.tsx

```diff
@@ -180,8 +190,34 @@ const CRUDDash: React.FC<CrudProps> = ({ firestore, data }) => {
           Volver al mapa
         </button>
       </div>
+
+      {/* Section for Download JSON and Analytics Link */}
+      <div className="absolute right-0 top-16 m-4">
+
+        {/* Download JSON Button */}
+        <button
+          onClick={handleDownload}
+          className="rounded-full bg-blue-500 px-4 py-2 text-white shadow-md hover:bg-blue-600"
+        >
+          Download JSON File
```

Comment:
This label should be "Guardar copia de datos".

Our app's UI language is Spanish. You can use Google Translate in the future for labels like this. Also remember, our users are not technical people. Instead of "Download JSON File" (scary acronym that doesn't mean anything to them), you could say "Save Database Backup" (they know that this is a database) or even "Save a copy of the data" (my translation)

Reply:
Oh right I was a bit confused as to what the admin panel was so I just went with instinct, probably should've cleared it out with you first before. I'll make the new changes and push them